### ISSUE TITLE
Can't resolve path in tutorial/install.jl

### ISSUE THREAD

**Issue by adigitoleo** [jpjones76/SeisIO.jl#76]
Copying the line from the README:
```
cd(dirname(pathof(SeisIO))); include("../tutorial/install.jl")
```
results in an error (Linux) because the trailing slash is not added to `pathof(SeisIO)`:
```
ERROR: LoadError: could not open file /home/leon/.julia/packages/SeisIO/xmXyu/tutorial../test/TestHelpers/0_check_deps.jl
```
The required diff for the tutorial/install.jl:
```
+ include(path * "/../test/TestHelpers/0_check_deps.jl")
- include(path * "../test/TestHelpers/0_check_deps.jl")
```

**Comment by tclements** [comment]
@adigitoleo I pushed a fix, we're hoping to get out a new version today.

**Comment by jpjones76** [comment]
I'll merge @tclements PR into the main branch once current automated tests conclude.

**Comment by jpjones76** [comment]
Merged and released with SeisIO v1.2.0. Please let me know that this is working, so I can close the issue.

**Comment by adigitoleo** [comment]
Yep it's working on v1.2 from github. Cheers.
